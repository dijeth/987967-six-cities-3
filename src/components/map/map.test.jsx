import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

const props = {
  centerCoord: [52.38333, 4.9],
  offersCoord: [
    [52.359385, 4.879898],
    [52.353995, 4.911789],
    [52.377303, 4.903075]
  ]
};

it(`<Map /> should be render correctly`, () => {
  const map = renderer.create(<Map {...props} />, {
    createNodeMock: () => {
      return document.createElement(`div`);
    }
  }).toJSON();

  expect(map).toMatchSnapshot();
});
