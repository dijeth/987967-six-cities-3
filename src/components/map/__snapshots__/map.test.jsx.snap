// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> should be render correctly 1`] = `
<section
  className="cities__map map"
>
  <div
    id="map"
    style={
      Object {
        "height": "100%",
      }
    }
  />
</section>
`;
